torch/_inductor/codegen/cpp.py

```diff
@@ -420,10 +508,11 @@ def codegen_loops(self, code, worksharing):
 
         if config.cpp.simdlen:
             # TODO(jansel): detect stride-1 dimension and vectorize that
-            if reductions:
+            if reductions and len(reductions.loops) > 0:
```

Comment:
bool(list) checks both None and empty, both of which would return False. So no need to add a len check here.

Reply:
Fixed.